scenarios/plugin.schema.README.md

```diff
@@ -0,0 +1,5 @@
+This file is generated by running the "plugins" module in the kraken project:
```

Comment:
Should we make a doc on how to write a plugin within krkn. I believe it was on the wolkenwalze github (now [arcaflow](https://github.com/arcalot/arcaflow-plugin-sdk-python)) but it is not outright mentioned in our docs here in this PR. Can be separate from this PR but I feel like this plugin schema command below is super important on how to generate it and without knowing to look in the folder (not out right mentioned in docs section) it might be hard to find this

Reply:
@paigerube14 thoughts on creating a github issue to add docs around plugins?